01_temperature/temperature.rb

```diff
@@ -0,0 +1,7 @@
+def ftoc(fahrenheit)
+   celsius=5.0/9*(fahrenheit-32)
```

Comment:
It isn't needed to assign the value to a variable (`celsius`). In Ruby methods return automatically the value that was returned from the last statement inside the method. [Here](http://ruby-for-beginners.rubymonstas.org/writing_methods/return_values.html) is an example. See how you can also improve this in the other PRs